rskj-core/src/test/java/org/ethereum/db/IndexedBlockStoreTest.java

```diff
@@ -430,7 +431,7 @@ public void test4() {
 
         DB indexDB = createMapDB(testDir);
 
-        KeyValueDataSource blocksDB = new LevelDbDataSource("blocks", config.databaseDir());
+        KeyValueDataSource blocksDB = new RocksDbDataSource("blocks", config.databaseDir());
```

Comment:
as we're not gonna replace LevelDb with RocksDb, at least, that this moment, but to support both, do you think we could have a parameterized tests here that create both `LevelDbDataSource` and `RocksDbDataSource`?

Reply:
Done